test/watershed.jl

```diff
@@ -0,0 +1,22 @@
+using ImageFiltering
+
+@testset "watershed" begin
+    
+    img = zeros(100, 100)
+    img = zeros(Float64, (100,100))
+    img[25:75, 25:75] = 1
+
+    img, _ = magnitude_phase(img)
```

Comment:
Sorry, I did a borked test and thought that this changed nothing. (I'd forgotten this is magnitude of gradient, not magnitude.)